src/index.js

```diff
@@ -0,0 +1,118 @@
+// ========= CHANGE TEMP =========
+
+const state = {
+  temp: 42,
+};
+
+const increaseTemp = (event) => {
+  state.temp += 1;
+  const tempDisplayed = document.querySelector('#temp-value');
+  tempDisplayed.textContent = state.temp;
+  if (state.temp >= 80) {
+    tempDisplayed.style.color = 'red';
+  } else if (state.temp >= 70) {
+    tempDisplayed.style.color = 'pink';
+  } else if (state.temp >= 60) {
+    tempDisplayed.style.color = 'orange';
+  } else if (state.temp >= 50) {
+    tempDisplayed.style.color = 'green';
+  } else if (state.temp <= 49) {
+    tempDisplayed.style.color = 'blue';
+  }
+  changeLandscape();
```

Comment:
Notice how much of this function is identical to the contents of `decreaseTemp`. This would be a great candidate to extract into its own method that can be called from both outer functions.